tests/Dockerfile.windows-x64-base

```diff
@@ -3,16 +3,18 @@ MAINTAINER Romain Beauxis <toots@rastageeks.org>
 
 RUN apt-get update && apt-get install -y --force-yes gawk aspcud
 
+WORKDIR /usr/src/mxe
+RUN git pull origin master
+RUN make cc
```

Comment:
Shouldn't this be fixed by updating the `dockcross/windows-x64` image used when building our base image? Also, it looks like we need to update the `gcc` depext:
```
Step 6/8 : RUN cd /usr/src/mxe/ && cat /home/opam/mxe-deps | xargs make
 ---> Running in 729fa8871f71
[pkg-list]  gcc
== gcc is now a dependency of virtual toolchain package cc
      - cc will build gcc, pkgconf, and other core toolchain packages
      - please update scripts accordingly (ignore if you are building gcc alone)
```
I'll take care of the later.

Reply:
Well, actually I don't know where this extdep is and also it looks like it's still okay to build `gcc` alone, which is prolly fine here.

Reply:
> Shouldn't this be fixed by updating the dockcross/windows-x64 image used when building our base image?

The latest dockcross/windows-x64 image is not recent enough. Specifically it fails to compile expat because the download links expired.